content/sensu-go/5.0/reference/rbac.md

```diff
@@ -1,132 +1,298 @@
+Users inherit all of the permissions from each role they are
+in.
 
-#### Attributes
+### Default roles
 
-username     | 
--------------|------ 
-description  | The name of the user. Cannot contain special characters.
-required     | true 
-type         | String
-example      | {{< highlight shell >}}"name": "admin"{{</highlight>}}
+Every [Sensu backend][1] includes:
 
-password     | 
--------------|------ 
-description  | The user's password. Cannot be empty. 
-required     | true 
-type         | String
-example      | {{< highlight shell >}}"password": "P@ssw0rd!"{{</highlight>}}
+- A global `admin` role that grants full access (read, create, update, and delete) to all [resource types][4] across namespaces
+- A global `read-only` role that grants read-only permission to all [resource types][4] across namespaces
 
-roles        | 
--------------|------ 
-description  | A list of roles the user belongs to.
-required     | false 
-type         | Array
-example      | {{< highlight shell >}}"roles": ["admin"]{{</highlight>}}
+### Viewing roles
 
-disabled     | 
--------------|------ 
-description  | The state of the user's account.
-required     | false 
-type         | Boolean 
-default      | false
-example      | {{< highlight shell >}}"disabled": false{{</highlight>}}
+You can use sensuctl to see a list of all roles within Sensu:
 
-### Group
+{{< highlight shell >}}
+sensuctl role list
+{{</highlight>}}
 
-Not available yet!
+To see the permissions and scope for a specific role:
 
-### Role
+{{< highlight shell >}}
+sensuctl role info admin
+{{</highlight>}}
 
-A role contains a set of rules, which represent permissions to Sensu resources.
-Roles can be assigned to one or multiple users. Each user can be a member of one
-or multiple roles. Users inherit all of the permissions from each role they are
-in.
+You can see that the default `admin` role has create, read, update, and delete access for all (`*`) [resource types][4] across all namespaces.
 
-The initial installation of Sensu includes an initial `admin` role that gives a
-user associated with it full access to the system.
+{{< highlight shell >}}
+Type   Namespace          Permissions         
+────── ──────────── ─────────────────────────── 
+*      *             create,read,update,delete  
+{{</highlight>}}
 
-#### Attributes
+### Creating a role
+
+You can use sensuctl to create a role.
+For example, the following command creates an admin role restricted to the ops namespace.
+
+{{< highlight shell >}}
+sensuctl role create ops-admin --create --read --update --delete --namespace ops --type *
+{{</highlight>}}
+
+You can also create a role using a JSON role definition.
+
+{{< highlight shell >}}
+{
+  "name": "ops-admin",
+  "namespace": "ops",
+  "rules": [
+    {
+      "verbs": ["create", "read", "update", "delete"],
+      "resources": ["*"],
+      "resourceNames": [""]
+    }
+  ]
+}
+{{</highlight>}}
+
+### Creating a cluster-wide role
```

Comment:
It would actually a ClusterRole/ClusterRoleBinding. I realize it's really confusing at first, let me know if you would like more details about that!

Reply:
Updated!